app/src/main/java/com/example/plantsapp/presentation/ui/utils/CalendarExt.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.plantsapp.presentation.ui.utils
+
+import java.util.Calendar
+import java.util.concurrent.TimeUnit
+
+fun Calendar.calculateDelay(
+    nextDayHour: Int = 0,
+    nextDayMinute: Int = 0,
+    nextDaySecond: Int = 0
+): Long {
+
+    val firstScheduledDate = (this.clone() as Calendar).apply {
```

Comment:
Lets talk about this tomorrow (it is about any other ways)
+ NOTE FOR ME: Will it work with REPLACE?